core/market/resolver/tests/sample/mod.rs

```diff
@@ -92,7 +92,7 @@ pub static POC_DEMAND_PROPERTIES_JSON: &str = r#"
   "golem.node.debug.subnet": "piotr",
   "golem.node.id.name": "test1",
   "golem.srv.comp.expiration": 1590765503361,
-  "golem.srv.comp.task_package": "hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://3.249.139.167:8000/rust-wasi-tutorial.zip"
+  "golem.srv.comp.task_package": "hash://sha3:D5E31B2EED628572A5898BF8C34447644BFC4B5130CFC1E4F10AEAA1:http://yacn2.dev.golem.network:8000/rust-wasi-tutorial.zip"
```

Comment:
Pls do not change the Resolver module at this stage. We are frozen.
And it is only a string. Not interpreted by resolver nor any1. This is a test only.

Reply:
So maybe it should contain some intentionally unimportant host name like `12.34.56.78`?

Reply:
From the resolver tests point of view, this is a bare string. You can put it `fusiek` :)
It does not need to be up to date. Its contents just mimics real values, that's all.

Reply:
Also, maybe in other files it's not important as well?

Reply:
yes. thats my point. Let's do not spend any more time on that, as this is of far importance